src/AbstractOperations/computed_field.jl

```diff
@@ -89,3 +89,62 @@ end
 
     @inbounds data[i′, j′, k′] = operand[i′, j′, k′]
 end
+
+struct FusedComputedFields
+    fields::Vector{ComputedField}
+    grid::AbstractGrid
+    datas::Vector{AbstractArray}
+    operands::Vector{AbstractOperation}
+    indices::Tuple
+
+    function FusedComputedFields(fields::Vector{ComputedField})
+        grid = fields[1].grid
+        indices = fields[1].indices
+        arch = architecture(fields[1])
+        sz = size(fields[1])
```

Comment:
We usually like to use `first` rather than `list[1]`, but that's a minor point...

Reply:
Updated.